$NetBSD: patch-pyproject.toml,v 1.3 2023/02/23 15:59:24 schmonz Exp $

Tarball lacks .git, so pass down version from our Makefile.

--- pyproject.toml.orig	2023-02-13 20:28:58.000000000 +0000
+++ pyproject.toml
@@ -8,7 +8,6 @@ module = [
   "lxml",
   "pytest",
   "requests_file",
-  "setuptools_scm",
   "unidecode",
   "xdg",
   "xdg.BaseDirectory",
@@ -16,7 +15,4 @@ module = [
 ignore_missing_imports = true
 
 [build-system]
-requires = ["setuptools>=45", "wheel", "setuptools_scm>=6.2"]
-
-[tool.setuptools_scm]
-write_to = "feed2exec/_version.py"
+requires = ["setuptools>=45", "wheel"]
